src/transformers/models/roformer/tokenization_roformer.py

```diff
@@ -166,22 +181,17 @@ def __getstate__(self):
 
     def __setstate__(self, d):
         self.__dict__ = d
-        try:
-            import rjieba
-        except ImportError:
-            raise ImportError(
-                "You need to install rjieba to use RoFormerTokenizer."
-                "See https://pypi.org/project/rjieba/ for installation."
-            )
-        self.jieba = rjieba
+        import jieba
+
+        self.jieba = jieba
```

Comment:
Same question here for @LysandreJik.

Reply:
Woul dbe nice to leave the try/except statement